ownCloud/Resources/Theming/generate.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# PREREQUISITE
+# Create symlink
+# ln -s SYNCDIR\ Config\ Data/5500Z000003poZfQAI/ios/current ~/Developer/github.com/owncloud/ios-app/ownCloud/Resources/Theming/theme.damken
+
+current_dir=$(pwd)
+theming_dir="/ownCloud/Resources/Theming"
+theme=$1
+
```

Comment:
I would add a check, if the script is executed in the app repository path. Otherwise script could fail, when the `current_dir` is not the root of the repository path:

`if [ -d "$theming_dir" ]; then`
`# add all lines below`
`fi`

Reply:
Sounds perfect 👍 I also thought about this. Could you commit in the branch?

Reply:
yes, I will do!